apps/scanner_worker/src/index.ts

```diff
@@ -181,7 +181,10 @@ const start = (): void => {
         });
 
         childProcess.stderr.on("data", (data) => {
-            console.log(`[${pid}] process: ${data}`);
+            // Filter empty lines
+            if (data.toString().trim() !== "") {
```

Comment:
Nit: Would it make sense to check on `.length == 0` for the trimmed string instead?

Reply:
It would probably have been an equally solid implementation.

Reply:
More explicitly, what I was after is: Does it make sense to micro-optimize for not constructing a temporary empty string instance? I'm asking this serious as I know little about the optimizations done by JavaScript runtimes.

Reply:
No idea of the performance difference, IMO no sense to try and micro-optimize it if there are no issues. The logs will be coming from ScanCode, so any micro-optimizations we'd be able to do here would be a drop in the bucket.